### PR TITLE
Remove startCase from card titles & Update Snapshots

### LINKED ISSUE #424: Punctuation and capital lettering missing on newsfeed.
## Acceptance Criteria

- [ ] Titles of articles, devotionals, sermons should display with all capitalization and punctuation.

## Context

Currently some titles are missing punctuation and the intermedial capitalization in NewSpring.

## UI Screenshot or Wireframe (optional)
screen shot 2018-03-22 at 1 25 42 pm

### PR DESCRIPTION
**Pull request by cdanwards**
Fixes #424 

## Creator

- [x] Build relevant tests if any apply
- [x] Ensure there are no new warnings
- [x] Upload an animated GIF showcasing the solution (if it applies)
- [x] Set a relevant reviewer

## Reviewer

- [x] Run `test` and make sure all tests pass
- [x] Review function and form on iOS
- [x] Review function and form on Android
- [ ] Review function and form on Web
- [x] Review new test logic

Per a conversation with our comms dept. we're going to opt to have the titles of articles reflect the titles that are originally in the CMS.

Also, this updates some snapshots that had not been previously updates.

### CODE REVIEW

File: src/@ui/FeedItemCard/index.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ const FeedItemCard = enhance(
         overlayColor={backgroundColor}
       />
       <StyledCardContent>
-        <H4>{startCase(toLower(title))}</H4>
+        <H4>{title}</H4>
```
Comment (burkeshartsis) [inline src/@ui/FeedItemCard/index.js]: Are titles capitalized correctly on Heighliner now?

Comment (cdanwards) [reply src/@ui/FeedItemCard/index.js]: @burkeshartsis Was there an issue with capitalization not correctly displaying beforehand? As far as I tested after making this change the titles displayed as they were entered into expressionengine.

Comment (conrad-vanl) [reply src/@ui/FeedItemCard/index.js]: it seems we started doing `startCase(toLower(` stuff here: https://github.com/NewSpring/Apollos/commit/a13f4f066ced5749acc74f20e23565b6a1869bae
> * updated title text to use title case
added card margins to space out cards in feed view while accounting to android quirks
attempted to get card image to be fluid. not happy with how this turned out.

@burkeshartsis after talking with @cdanwards on this it appears the content team wants titles to be formatted how they are (regarding capitalization) in expression engine. We are overwriting that format.

**Comment by conrad-vanl** [comment]
Removing myself as a reviewer on this. I believe @burkeshartsis is best equipped to review this one with @cdanwards

File: src/@ui/FeedItemCard/index.js (hunk 1 above)
Comment (burkeshartsis) [reply src/@ui/FeedItemCard/index.js]: I am from the school of "never trust a copy writer." But am happy to remove the `startCase(toLower(` references from the app. I've always felt like this was something that should happen on Heighliner anyways. Let's keep the isolated fix for `FeedItemCard` here and we can track the rest of the refactor in this new issue (#580).

**Comment by richarddubay** [comment]
I think that if the capitalization patterns are off after this change then that means that our content team has some work to do in order to make them right.

**Comment by burkeshartsis** [comment]
@richarddubay probably which goes back to, "never trust a copywriter to..."

**Comment by richarddubay** [comment]
@IsaacHardy You might want to notify the content team that this change is coming...just in case they feel like they have some stuff to fix before we push this to production.

**Pull request merged**